src/vm/jitinterface.cpp

```diff
@@ -6827,15 +6827,23 @@ DWORD CEEInfo::getMethodAttribsInternal (CORINFO_METHOD_HANDLE ftn)
         result |= CORINFO_FLG_DONT_INLINE_CALLER;
     }
 
-    // Check for the aggressive optimization directive. AggressiveOptimization only makes sense for IL methods.
+    // Somne flags only need to be set for IL methopds
     DWORD ilMethodImplAttribs = 0;
     if (pMD->IsIL())
     {
+        // Check for the aggressive optimization directive. 
+        // AggressiveOptimization only makes sense for IL methods.
         ilMethodImplAttribs = pMD->GetImplAttrs();
         if (IsMiAggressiveOptimization(ilMethodImplAttribs))
         {
             result |= CORINFO_FLG_AGGRESSIVE_OPT;
         }
+        
+        if (!pMD->IsVersionableWithoutJumpStamp())
```

Comment:
`getMethodAttribsInternal` is called for every callsite. We do not need this flag for every callsite. It is unnecessary overhead to compute this flag here. We need this flag once per method JITed only. This should be `JIT_FLAG` instead.

Reply:
This has now been removed